frontline-functions-quickstart/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "frontline-functions-quickstart",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "@twilio-labs/runtime-helpers": "^0.1.2"
```

Comment:
You'll have to specify in this `package.json` the `twilio` dependency as well. The top level `package.json` is only used for local tests. This file is the one that is used to install dependencies for the customer